zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+
+This model allows for the community to guide its evolution and possible replacement via Governance Decisions, and so would be activated indefinitely.
+
+
+# Motivation
+
+Zcash governance must represent voices from all over the world, who need digital cash for their security and for their dignity. It must not be dependent on individuals who, despite their best intentions, are flawed and prone to temptations to control and coerce. It must be durable and withstand attempted capture of all forms.
+
+zBloc is a governance model that is designed for decentralization and accountability, to include more voices and ensure results from those who receive Protocol-defined Ecosystem Funding.
+
+It is built to allow new innovative and positive voices to rise up and carry the flag of the Zcash community forward, in recognition of the risk that any one group may become captured or ineffective.
+
+
+# Requirements
+
+* This ZIP covers how to decide on Proposals for potential changes to Zcash Governance and/or disbursement of Protocol-defined Ecosystem Funding.
+* There is a well-defined, publically agreed, process for evaluation and community feedback on both Governance Proposals and Grant Proposals.
+* Funds are held in a multisig resistant to compromise of some of the parties' keys, up to a given threshold.
+* The voting mechanism used for decision-making is similarly resistant to compromise of some of the parties' keys, up to a given threshold.
+* No single party's non-cooperation or loss of keys is able to cause any Protocol-defined Ecosystem Funding to be locked up unrecoverably.
+* 80% of the block rewards are given to miners, and 20% to a Zcash Community Fund seeded from the Deferred Dev Fund Lockbox.
+* The funds from the 20% funding stream will be usable immediately.
+* The funds in the Deferred Dev Fund Lockbox will be usable once a mechanism is available for disbursing them.
+* Any use of Deferred Dev Fund Lockbox funds is consistent with the purpose specified in [^zip-1015-lockbox] of "funding grants to ecosystem participants".
+
+
+# Non-requirements
+
+Any changes to the ZCG or its governance, such as its expansion to include more members, may be desired but are specifically outside this proposal’s scope.
+
+The full implementation of this proposal would require the design and implementation of a lockbox distribution mechanism, likely using multisig. This mechanism would be similar to that needed for an alternative proposal, the Community and Coin-holder Funding Model [^draft-ecc-community-and-coinholder]. However, this proposal does not require that a lockbox distribution mechanism be implemented at the same time as a change to the funding stream.
```

Comment:
This gets to my comment above: any of the ecc-community-and-coinholder options would work for this proposal as well.